automation/system_test/prepare.py

```diff
@@ -326,10 +316,6 @@ def _patch_mlrun(self, provctl_path):
         )
         mlrun_archive = f"./mlrun-{self._mlrun_version}.tar"
 
-        repo_arg = ""
-        if self._override_full_image_repo:
-            repo_arg = f"--override-image-pull-repo {self._override_full_image_repo}"
-
```

Comment:
Why is this (and the part where it is passed to provctl) removed?

Reply:
Generally - no real reason to use this flag in provctl, we're anyway passing the images to `--override-images ` already with the right registry and repo, passing also this flag doesn't give anything (it just makes provctl override the registry and repo, to the same registry and repo)
Specifically - 
For scenario 3, I need to work against a docker repo which is not `mlrun` (and also possibly not `ghcr.io/`) but practically no one will ever really want to change the mlrun-ui tested code, therefore you can see that I'm always taking mlrun-ui from upstream, but based on what was asked, decide where to take from mlrun-api, therefore mlrun-ui and mlrun-api may not be from the same registry+repo, so with `--override-images ` I can do it, and it's working, but if we're adding `--override-image-pull-repo` it restrict them to be from the same registry+repo

Reply:
👍